File: src/SME.SGP.Dto/DisciplinaDto.cs
namespace SME.SGP.Dto
{
    public class DisciplinaDto
    {
        public DisciplinaDto()
        {
            Regencia = false;
        }

        public int CodigoComponenteCurricular { get; set; }
        public string Nome { get; set; }
        public bool PossuiObjetivos { get; set; }
        public bool Regencia { get; set; }
    }
}

File: src/SME.SGP.WebClient/src/dtos/disciplinaDto.js
export default class DisciplinaDTO {
  constructor(codigo, nome, possuiObjetivos, regencia) {
    this.codigo = codigo;
    this.nome = nome;
    this.possuiObjetivos = possuiObjetivos;
    this.regencia = regencia;
    this.selecionada = false;
  }
}
